src/alfasim_sdk/_internal/alfacase/generate_case_description_docstring.py

```diff
@@ -86,7 +90,7 @@ def generate_list_of_units(category: str) -> str:
         f".. admonition:: Available units for category '{category}'",
         "    :class: dropdown",
         "",
-        *sorted(body, key=str.casefold),
+        *sorted(body),
```

Comment:
```suggestion
        *sorted(body, key=lambda x: (x.casefold(), x)),
```

Reply:
Thanks for the careful review, it's all done!

Reply:
`*sorted(body, key=lambda x: (x.casefold(), x.swapcase())),` will put the ones with lower case first, but I guess that is marginal change =)

Reply:
I'm with you, having lowercase letters first is better presentable. It's done.